release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2139,6 +2158,11 @@ $ ./openshift-install wait-for install-complete
 
 * When using the Technology Preview OCI feature with the oc-mirror CLI plug-in, the mirrored catalog embeds all of the Operator bundles, instead of filtering only on those specified in the image set configuration file. (link:https://issues.redhat.com/browse/OCPBUGS-5085[*OCPBUGS-5085*])
 
+* Fully populated load balancers on {rh-openstack} that are created with the OVN driver can contain pools that are stuck in a pending creation status. This issue can cause problems for clusters that are deployed on {rh-openstack}. To resolve the issue, update your {rh-openstack} packages. link:https://bugzilla.redhat.com/show_bug.cgi?id=2042976[*BZ#2042976*]
+
+* Bulk load-balancer member updates on {rh-openstack} can return a 500 code in response to PUT requests. This issue can cause problems for clusters that are deployed on {rh-openstack}. To resolve the issue, update your {rh-openstack} packages. link:https://bugzilla.redhat.com/show_bug.cgi?id=2100135[*BZ#2100135*]
```

Comment:
This only affects OSP17, should we say that?

Reply:
@dulek Both, or just L2163? 

```suggestion
* Fully populated load balancers on {rh-openstack} 17 that are created with the OVN driver can contain pools that are stuck in a pending creation status. This issue can cause problems for clusters that are deployed on {rh-openstack}. To resolve the issue, update your {rh-openstack} packages. link:https://bugzilla.redhat.com/show_bug.cgi?id=2042976[*BZ#2042976*]

* Bulk load-balancer member updates on {rh-openstack} 17 can return a 500 code in response to PUT requests. This issue can cause problems for clusters that are deployed on {rh-openstack}. To resolve the issue, update your {rh-openstack} packages. link:https://bugzilla.redhat.com/show_bug.cgi?id=2100135[*BZ#2100135*]
```

Reply:
Aww, no, I'm wrong, it's all good as is. Sorry, so many bugs I got confused, this comment thread can be disregarded.